test/vm.install

```diff
@@ -7,6 +7,11 @@ if grep -q 'ID=debian' /usr/lib/os-release; then
     echo '* soft core unlimited' >> /etc/security/limits.conf
 fi
 
+# for opensuse images, allow dmesg to read /var/lib/libvirt/pxe-config
```

Comment:
Certainly not dmesg, but dnsmasq? This looks like an OS bug workaround -- please file a suse bug and link it here as a comment, with a `# HACK:` prefix (our usual thing to grep for, we regularly want to clean these up).

Reply:
My bad for the mis-wording. As for actual fix, the App Armour configs for SUSE only give dnsmasq (speaking in the context of libvirt) access to /var/lib/libvirt/dnsmasq from what I've seen pxe-config isn't an standard path and from digging around cockpit I haven't seen any references to this that would create this issue under production.

The path is only defined in tests here: https://github.com/cockpit-project/cockpit-machines/blob/ffbbf25e527d642508194c0ef993bde5c8359212/test/machinesxmls.py#L103 since it's a constant I thought this would be the more correct action to fix the issue. Still I'm happy to make any changes if you've got an alternative method you'd prefer for this

Reply:
Thanks for the explanation! I sent PR #1658, if that goes green and avoids that hack, let's do that instead? If not, ok.